### PR TITLE
build: update Angular tooling packages to `15.0.0-next.3`

### PR DESCRIPTION
**Pull request by alan-agius4**
- This update is needed to implement the changes in `ng add localize` https://github.com/angular/angular/pull/47569
- Add missing `root` options to all `angular.json`, this is required as otherwise the angular.json validation will fail.
- Remove `require.context` from test.ts integration test, as this is no longer needed.
- Update payloads golden files.

### CODE REVIEW

File: integration/bazel_workspace_tests/bazel_ngtsc_plugin/test/e2e/BUILD.bazel [hunk 1]
```diff
@@ -26,6 +26,7 @@ ts_library(
 
 protractor_web_test_suite(
     name = "test",
+    browsers = ["@npm//@angular/build-tooling/bazel/browsers/chromium:chromium"],
```
Comment (devversion) [inline integration/bazel_workspace_tests/bazel_ngtsc_plugin/test/e2e/BUILD.bazel:29]: can't the integration test use the rules webtesting browser? I think it shouldn't use the private build tooling from dev-infra. not sure what the latest version there is

File: goldens/size-tracking/integration-payloads.json [hunk 2]
```diff
@@ -33,22 +33,22 @@
...
+      "main": 223752,
       "polyfills": 33842,
       "src_app_lazy_lazy_routes_ts": 487
     }
   },
   "forms": {
     "uncompressed": {
       "runtime": 1060,
-      "main": 157136,
+      "main": 155712,
```
Comment (AndrewKushnir) [inline goldens/size-tracking/integration-payloads.json:44]: This is awesome 🎉 @alan-agius4 do we know which change caused the payload size decrease?

**Review by AndrewKushnir — APPROVED** [review]

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: size-tracking

File: integration/npm_package_archives.bzl [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # The @npm packages at the root node_modules are used by integration tests
 # with `file:../../node_modules/foobar` references
 NPM_PACKAGE_ARCHIVES = [
+    "@angular/build-tooling",
```
Comment (devversion) [inline integration/npm_package_archives.bzl:4]: This should not be done I feel like. Integration tests should not rely on the dev-infra private build tooling- as integration tests mostly verify user scenarios

**Review by devversion — APPROVED** [review]

File: goldens/size-tracking/integration-payloads.json (hunk 2 above)
Comment (alan-agius4) [reply goldens/size-tracking/integration-payloads.json:44]: We now output es2022 by default and only downlevel features that are needed to be downlevelled based on the supported browsers.

File: integration/bazel_workspace_tests/bazel_ngtsc_plugin/test/e2e/BUILD.bazel (hunk 1 above)
Comment (alan-agius4) [reply integration/bazel_workspace_tests/bazel_ngtsc_plugin/test/e2e/BUILD.bazel:29]: As per offline convo, let’s delete its

**Review by JiaLiPassion — APPROVED** [review]
Reviewed-for: zone.js
LGTM

**Review by JiaLiPassion — APPROVED** [review]
Reviewed-for: zone.js
LGTM

**Review by JiaLiPassion — APPROVED** [review]
LGTM

**Comment by alan-agius4** [comment]
**Caretaker:** Github is not allowing me to remove the pending reviewers even though the current approvals suffice.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit ce59f1a14dc61766dc9ff4600ad52f53e6fce209.

**Pull request closed without merging**